routing/index_graph_starter.cpp

```diff
@@ -135,12 +197,29 @@ void IndexGraphStarter::FindPointsWithCommonFeature(Joint::Id jointId0, Joint::I
   bool found = false;
   double minWeight = -1.0;
 
+  // Additional edges.
+  if (m_jointsOfAdditionalEdges.count(jointId0) != 0 && m_jointsOfAdditionalEdges.count(jointId1) != 0)
```

Comment:
In any case, why not to modify "ForEachPoint" to give you RoadPoints from fake features too? In this case you may completely remove this new block of code, right?